asv_bench/benchmarks/strings.py

```diff
@@ -87,6 +84,29 @@ def time_repeat(self, repeats):
         self.s.str.repeat(self.repeat)
 
 
+class Cat(object):
+
+    goal_time = 0.2
+    params = ([None, 5], [None, ','], [None, '-'], [0.0, 1e-4, 0.1])
+    param_names = ['others', 'sep', 'na_rep', 'na_frac']
+
+    def setup(self, others, sep, na_rep, na_frac):
+        N = int(5e5)
```

Comment:
Can you see how we do this in other benchmarks? I think we usually just pick 100_000 so for consistency might be nice to reflect that here

Reply:
No problem, was trying to get the results near `goal_time`, but maybe I'm misinterpreting what that is supposed to be